pkg/kn/commands/revision/revision_list.go

```diff
@@ -66,3 +86,26 @@ func NewRevisionListCommand(p *commands.KnParams) *cobra.Command {
 	revisionListFlags.AddFlags(revisionListCommand)
 	return revisionListCommand
 }
+
+func getRevisionInfo(args []string, namespace string, client serving.ServingV1alpha1Interface) (*v1alpha1.RevisionList, error) {
+
+	switch len(args) {
+	case 0:
+		revision, err := client.Revisions(namespace).List(v1.ListOptions{})
```

Comment:
case 0 and 1 are the same except list options. seems like it would be better to have single client.Revisions(...) call and have listOpts variable be set in switch statement.

Reply:
Updated